### PR TITLE
Runtime common

### PR DESCRIPTION
**Pull request by jiguantong**

### CODE REVIEW

**Comment by AurevoirXavier** [comment]
If you merge the main, the CI problems will be gone.

File: runtime/common/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+# crates.io
+array-bytes    = { version = "4.1" }
+codec          = { package = "parity-scale-codec", version = "3.1", default-features = false, features = ["derive"] }
+ethabi         = { version = "17.0", default-features = false }
+ethereum       = { version = "0.12", default-features = false, features = ["with-codec"] }
+ethereum-types = { version = "0.13.1", default-features = false, features = ["rlp", "codec"] }
```
Comment (AurevoirXavier) [inline runtime/common/Cargo.toml:18]: Please check the `[dependencies]`.

Found a lot of unused dependencies.

**Review by AurevoirXavier — CHANGES_REQUESTED** [review]
Please resolve the review comments.

File: node/Cargo.toml [hunk 2]
```diff
@@ -123,3 +123,4 @@ rococo-native = [
 	# polkadot
 	"polkadot-cli/rococo-native",
 ]
+
```
Comment (AsceticBear) [inline node/Cargo.toml:126]: A nitpick. Empty line

File: runtime/common/Cargo.toml [hunk 3]
```diff
@@ -0,0 +1,28 @@
+[package]
+authors     = ["Darwinia Network <hello@darwinia.network>"]
+description = "Darwinia Common Runtime"
+edition     = "2021"
+homepage    = "https://darwinia.network/"
+license     = "GPL-3.0"
+name        = "darwinia-common-runtime"
+readme      = "README.md"
+repository  = "https://github.com/darwinia-network/darwinia-parachain"
```
Comment (AsceticBear) [inline runtime/common/Cargo.toml:9]: Should be `repository  = "https://github.com/darwinia-network/darwinia"`

File: runtime/common/src/xcm_barrier.rs [hunk 4]
```diff
@@ -20,16 +20,19 @@
...
 
 frame_support::match_types! {
 	pub type ParentOrParentsExecutivePlurality: impl Contains<MultiLocation> = {
 		MultiLocation { parents: 1, interior: Here } |
 		MultiLocation { parents: 1, interior: X1(Plurality { id: BodyId::Executive, .. }) }
 	};
+	pub type ParentOrSiblings: impl Contains<MultiLocation> = {
+		MultiLocation { parents: 1, interior: Here } |
+		MultiLocation { parents: 1, interior: X1(_) }
+	};
```
Comment (AsceticBear) [inline runtime/common/src/xcm_barrier.rs:35]: @AurevoirXavier Please help review this. I am okay with other changes.

**Review by AsceticBear — APPROVED** [review]

**Pull request merged**